src/loggingFunctions.js

```diff
@@ -0,0 +1,106 @@
+var logUtil = require('./logUtil');
+var linesUtil = require('./linesUtil');
+var config = require('./config');
+
+module.exports = [
+	//////////////////
+	// main methods //
+	//////////////////
+
+	{	//default

```

Comment:
doing it this way, we can make sure we handle all of our log functions the same.